samples/processing/kmeans_preprocessing.py

```diff
@@ -0,0 +1,24 @@
+import pickle
```

Comment:
add to pre-reqs - upload this script to an S3 bucket and say the path needs to be updated in 
```
# The input path of our source code
s3URI: s3://<YOUR BUCKET>/path
```